voting/tests/common/scenarios.rs

```diff
@@ -317,3 +330,77 @@ pub fn test_vote(ctx: &RwLock<Context>) {
 
     //FIXME: `InvalidVoterSignature` should be checked after it is fully implemented.
 }
+
+pub fn test_publish_result(ctx: &RwLock<Context>){
+    for stateful in ctx.write().statefuls.values_mut() {
+        stateful.set_storage(ServiceRef::create_export(Box::new(MockDb::default()) as Box<dyn SubStorageAccess>))
+    }
+
+    let admin: Ed25519KeyPair = Random.generate().unwrap();
+    let admin_key = admin.public();
+
+    ctx.write()
+        .init_genesises
+        .get_mut("general_meeting")
+        .unwrap()
+        .init_genesis(&serde_cbor::to_vec(&admin_key).unwrap());
+
+    let create_meeting = tx_create_general_meeting(&admin.public(), &admin.private(), 2, 12, 18);
```

Comment:
It's hard to know which is the timestamp.
How about declaring good names to the timestamps used in this test?